src/components/SelectBox.js

```diff
@@ -43,16 +44,26 @@ class SelectBox extends React.Component {
       this.props.onChange(id)
     }
   }
+  getValidationState () {
```

Comment:
Add a blank line between component methods.

I'd just call it `getValidationClass`.